package/MDAnalysis/core/universe.py

```diff
@@ -629,22 +659,34 @@ def select_atoms(self, *args, **kwargs):
 
     @property
     def bonds(self):
-        """Bonds between atoms"""
+        """Bonds between atoms.
+
+        :meta private:
+        """
         return self.atoms.bonds
 
     @property
     def angles(self):
-        """Angles between atoms"""
+        """Angles between atoms.
+
+        :meta private:
```

Comment:
I wonder if, if the `angles` attribute was removed in the class docstring, would this one be able to be used and not private?

Reply:
This is already answered in https://github.com/MDAnalysis/mdanalysis/pull/3438#issuecomment-958421622, sorry

Reply:
I don't think so.

Reply:
... I mean, as long as it still contains ":meta private:". If you remove the one in the class doc string then this one can be used and it will be auto-documented if it does _not_ contain ":meta private:".